tests/unit/testsuite.js

```diff
@@ -339,4 +339,23 @@ window.domEqual = function( selector, modifier, message ) {
 	}
 };
 
+// Assertation for checking that an element has multiple classes
+QUnit.assert.hasClasses = function( element, classes ) {
+	var result,
+		classArray = classes.split( " " ),
+		missing = [],
+		found = [];
+
+	classArray.forEach( function( value ) {
+		if ( !element.hasClass( value ) ) {
+			missing.push( value );
+		} else {
+			found.push( value );
+		}
+	});
+	result = !missing.length;
+	QUnit.push( result, classes, result ? classes : found.join( " " ),
```

Comment:
Yes thats correct ill swap these. Also the ternary here is not needed, if everything is found `found.join( " " ) === classes` so its pointless. It can just be `found.join( " " )`, which will always be what we want here.